constellation/docker_util.py

```diff
@@ -120,6 +120,23 @@ def container_wait_running(container, poll=0.1, timeout=1):
     return container
 
 
+def container_remove_wait(container, poll=0.1, timeout=1):
```

Comment:
Where is this used?

Reply:
In the hint-deploy PR [see here](https://github.com/mrc-ide/hint-deploy/pull/16/files#diff-6a8323646cb68579d92f9b9064e2ed3eR152-R156) - this is needed to avoid naming collisions when replacing containers